Packs/FeedMISP/Integrations/FeedMISP/FeedMISP.py

```diff
@@ -335,6 +337,19 @@ def update_indicators_iterator(indicators_iterator: List[Dict[str, Any]],
     return []
 
 
+def search_query_indicators_pagination(client: Client, params_dict: Dict[str, Any]) -> Dict[str, Any]:
+    params_dict['page'] = 0
+    response: Dict[str, Dict[str, List]] = {'response': {'Attribute': []}}
+    while True:
```

Comment:
Lets remove the while True and replace it to a condition relates to the len of _search_query_per_page_ var.